ecs-web/main.tf

```diff
@@ -128,51 +138,54 @@ data "template_file" "concourse_basic_auth" {
 { "name": "CONCOURSE_ADD_LOCAL_USER", "value": "$${concourse_auth_username}:$${concourse_auth_password}" },
 EOF
 
-  vars {
-    concourse_auth_username = "${var.concourse_auth_username}"
-    concourse_auth_password = "${var.concourse_auth_password}"
-  }
+
+vars = {
+concourse_auth_username = coalesce(var.concourse_auth_username, 0)
+concourse_auth_password = coalesce(var.concourse_auth_password, 0)
+}
 }
 
 data "template_file" "concourse_basic_auth_main_team_local_user" {
-  template = <<EOF
+template = <<EOF
 { "name": "CONCOURSE_MAIN_TEAM_LOCAL_USER", "value": "$${concourse_auth_username}" },
 EOF
 
-  vars {
-    concourse_auth_username = "${var.concourse_auth_main_team_local_user}"
-  }
+
+vars = {
```

Comment:
Indentation is also wrong here. For some reason the `upgrade.0.12` command messes up with the indentation around `EOF` blocks